Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableThemeProperty.cs

```diff
@@ -16,20 +17,43 @@ public class InteractableThemeProperty
         public InteractableThemePropertyValueTypes Type;
         public List<InteractableThemePropertyValue> Values;
         public InteractableThemePropertyValue StartValue;
-        public int PropId;
+        public int PropId; // i.e OptionIndex
         public List<ShaderProperties> ShaderOptions;
         public List<string> ShaderOptionNames;
         public InteractableThemePropertyValue Default;
         public string ShaderName;
 
-        public string GetShaderPropId()
+        private List<int> ShaderPropertyIDs = null;
+        private const string DefaultProperty = "_Color";
+
+        public int GetShaderPropertyId()
```

Comment:
Nit: Can you also add doccomments to these functions (i.e. maybe recommending the int flavor over the string flavor?).

Reply:
Documentation comments added